### PR TITLE
rename some "claim" instances to 'reclaim"

### PR DESCRIPTION
**Pull request by geoknee**

### CODE REVIEW

File: nitro-protocol/contracts/interfaces/IMultiAssetHolder.sol [hunk 1]
```diff
@@ -50,7 +50,7 @@ interface IMultiAssetHolder {
      * @param targetOutcomeBytes The abi.encode of target channel outcome
      * @param targetAssetIndex the index of the targetted asset in the target outcome.
      */
-    struct ClaimArgs {
+    struct RereclaimArgs {
```
Comment (nksazonov) [inline nitro-protocol/contracts/interfaces/IMultiAssetHolder.sol:53]:
```suggestion
    struct ReclaimArgs {
```

File: nitro-protocol/contracts/interfaces/IMultiAssetHolder.sol [hunk 2]
```diff
@@ -64,9 +64,9 @@ interface IMultiAssetHolder {
     /**
      * @notice Reclaim moves money from a target channel back into a ledger channel which is guaranteeing it. The guarantee is removed from the ledger channel.
      * @dev Reclaim moves money from a target channel back into a ledger channel which is guaranteeing it. The guarantee is removed from the ledger channel.
-     * @param claimArgs arguments used in the claim function. Used to avoid stack too deep error.
+     * @param reclaimArgs arguments used in the claim function. Used to avoid stack too deep error.
      */
-    function reclaim(ClaimArgs memory claimArgs) external;
+    function reclaim(RereclaimArgs memory reclaimArgs) external;
```
Comment (nksazonov) [inline nitro-protocol/contracts/interfaces/IMultiAssetHolder.sol:69]:
```suggestion
    function reclaim(ReclaimArgs memory reclaimArgs) external;
```

File: nitro-protocol/contracts/MultiAssetHolder.sol [hunk 3]
```diff
@@ -381,25 +381,25 @@ contract MultiAssetHolder is IMultiAssetHolder, StatusManager {
      * @dev Updates the fingerprint of the outcome for the source channel and emit an event for it.
      */
     function _apply_reclaim_effects(
-        ClaimArgs memory claimArgs,
+        RereclaimArgs memory reclaimArgs,
```
Comment (nksazonov) [inline nitro-protocol/contracts/MultiAssetHolder.sol:384]:
```suggestion
        ReclaimArgs memory reclaimArgs,
```

File: nitro-protocol/contracts/MultiAssetHolder.sol [hunk 4]
```diff
@@ -244,36 +244,36 @@ contract MultiAssetHolder is IMultiAssetHolder, StatusManager {
...
 
-        _apply_reclaim_effects(claimArgs, sourceOutcome, newSourceAllocations);
+        _apply_reclaim_effects(reclaimArgs, sourceOutcome, newSourceAllocations);
     }
 
     /**
      * @dev Checks that the source and target channels are finalized; that the supplied outcomes match the stored fingerprints; that the asset is identical in source and target. Computes and returns the decoded outcomes.
      */
-    function _apply_reclaim_checks(ClaimArgs memory claimArgs)
+    function _apply_reclaim_checks(RereclaimArgs memory reclaimArgs)
```
Comment (nksazonov) [inline nitro-protocol/contracts/MultiAssetHolder.sol:276]:
```suggestion
    function _apply_reclaim_checks(ReclaimArgs memory reclaimArgs)
```

File: nitro-protocol/contracts/MultiAssetHolder.sol [hunk 5]
```diff
@@ -244,36 +244,36 @@ contract MultiAssetHolder is IMultiAssetHolder, StatusManager {
     /**
      * @notice Reclaim moves money from a target channel back into a ledger channel which is guaranteeing it. The guarantee is removed from the ledger channel.
      * @dev Reclaim moves money from a target channel back into a ledger channel which is guaranteeing it. The guarantee is removed from the ledger channel.
-     * @param claimArgs arguments used in the claim function. Used to avoid stack too deep error.
+     * @param reclaimArgs arguments used in the reclaim function. Used to avoid stack too deep error.
      */
-    function reclaim(ClaimArgs memory claimArgs) external override {
+    function reclaim(RereclaimArgs memory reclaimArgs) external override {
```
Comment (nksazonov) [inline nitro-protocol/contracts/MultiAssetHolder.sol:249]:
```suggestion
    function reclaim(ReclaimArgs memory reclaimArgs) external override {
```

**Review by nksazonov — APPROVED** [review]

**Comment by geoknee** [comment]
🤦 thanks for checking @nksazonov !

**Pull request merged**